### PR TITLE
chore: upgrade to snapshot-metrics 1.1.0

### PR DESCRIPTION
**Pull request by wa0x6e**
## Changes 

- Update snapshot-sentry to latest (add support for JSON-RPC error)
- Update snapshot-metrics to latest (add support for pushgateway, and embedded `express_open_connections_size` metrics)

### CODE REVIEW

File: src/metrics.ts [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
 import gateways from './gateways.json';
 import { providersMap, IMAGE_PROVIDERS, JSON_PROVIDERS } from './providers/utils';
 
-let server;
-
 export default function initMetrics(app: Express) {
   init(app, {
     normalizedPath: [['^/ipfs/.*', '/ipfs/#url']],
-    whitelistedPath: [/^\/$/, /^\/upload$/, /^\/ipfs\/.*$/]
+    whitelistedPath: [/^\/$/, /^\/upload$/, /^\/ipfs\/.*$/],
```
Comment (Todmy) [inline src/metrics.ts:10]: why do we need this line? What paths do you want to omit?

**Review by Todmy — CHANGES_REQUESTED** [review]

File: src/metrics.ts (hunk 1 above)
Comment (wa0x6e) [reply src/metrics.ts:10]: Everything not caught by the regex

Comment (Todmy) [reply src/metrics.ts:10]: Which routes do we have here except `/`, `/upload` and `/ipfs`?

Comment (Todmy) [reply src/metrics.ts:10]: Which routes do we have here except `/`, `/upload` and `/ipfs`?

**Review by Todmy — CHANGES_REQUESTED** [review]

File: src/metrics.ts (hunk 1 above)
Comment (wa0x6e) [reply src/metrics.ts:10]: None. It's to avoid catching the fallback routes

**Review by Todmy — APPROVED** [review]
utACK

**Pull request merged**